docs/developer/README.md

```diff
@@ -62,6 +62,20 @@ Linters:
 docker-compose exec app ./manage.py lint
 ```
 
+## Accessibility Scanning
+
+The tool `pa11y-ci` is used to scan pages for compliance with a set of
+accessibility rules. The scan runs as part of our CI setup (see
+`.github/workflows/test.yaml`) but it can also be run locally. To run locally,
+just
```

Comment:
I'd strike the word "just" or replace it with "type"/"use". Good to avoid words like "just", "simply", "obviously", "easily" in documentation where possible.